README.md

```diff
@@ -129,108 +129,10 @@ To shut down, the whole `docker-compose.yml` file and all services inside it the
-
-### Dependency management
-
-We are following [Yarn](https://yarnpkg.com/) package manager.
-
-Our `package.json` file is gitignored to enforce strict management of our client dependencies.
-
-In order to add a new dependency, please:
-
-1. use `yarn add <your-package>`
-2. `git checkout -b dependency/<your-package>`
-3. `git add --force flint.ui/package.json`
-4. `git push` and submit a PR with only the `package.json` changes
-
-Our CI build will test for conflicts and your new dependency will be added after your PR has been reviewed.
-
-### Code Style
-
-The Vue.js style guide has been used to maintain a consistent style throughout the project. Click [here](https://vuejs.org/v2/style-guide/) to learn more about the style guide.
-
-One exception is the case used to describe the ecological models exposed to our FLINT client. Because model names, and sometimes model variables, often use acronyms these remain capitalized.
-
-Obvious examples are:
-
-- GCBM: Generic Carbon Budget model
-- FLINT: Full Lands Integration Tool
-- RothC: Rothamstead Carbon model
-
-We recommend using ESLint and the Prettier plugins to make conforming to the Vue style guide easy. There is official integration for VSCode and other editors. Find out more [here](https://eslint.vuejs.org/user-guide/).
-
-If you have ESLint installed globally, you can run it from the `flint.ui` directory. For example, this will detect any errors in the `src` folder:
-
-```shell
-eslint --ext .js,.vue src
-```
-
-Minor errors can be fixed automatically:
-
-```shell
-eslint --fix --ext .js,.vue src
-```
-
-## Code Style
-
-- The Vue.js style guide has been used to maintain a consistent style throughout the project. Click [here](https://vuejs.org/v2/style-guide/) to learn more about the style guide.
-
-## How to Get Involved?
-moja global welcomes a wide range of contributions as explained in [Contributing document](https://github.com/moja-global/About-moja-global/blob/master/CONTRIBUTING.md) and the [About moja-global Wiki](https://github.com/moja-global/.github/wiki).
-
-## FAQ and Other Questions
-- You can find FAQs on the [Wiki](https://github.com/moja.global/.github/wiki).
-- If you have a question about the code, submit [user feedback](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Provide-User-Feedback.md) in the relevant repository
-- If you have a general question about a project or repository or moja global, [join moja global](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Join-moja-global.md) and
-  - [submit a discussion](https://help.github.com/en/articles/about-team-discussions) to the project, repository or moja global [team](https://github.com/orgs/moja-global/teams)
-  - [submit a message](https://get.slack.help/hc/en-us/categories/200111606#send-messages) to the relevant channel on [moja global's Slack workspace](mojaglobal.slack.com).
```

Comment:
Hey, could you please bring back this part?

Reply:
Sure